universite-du-quebec-a-montreal.csl

```diff
@@ -31,20 +31,33 @@
     <terms>
       <term name="editortranslator" form="short">dir. et trad.</term>
       <term name="editor" form="short">dir.</term>
-      <term name="translator" form="short">trad.</term>
+      <term name="translator" form="short">
+        <single>trad.</single>
+        <multiple>trad.</multiple>
+      </term>
       <term name="no date" form="short">s. d.</term>
       <term name="in">Dans</term>
       <term name="retrieved">Récupéré de</term>
       <term name="presented at">Communication présentée à</term>
-      <term name="page" form="short">p.</term>
+      <term name="page" form="short">
+        <single>p.</single>
+        <multiple>p.</multiple>
+      </term>
       <term name="in press">sous presse</term>
       <term name="director" form="short">réal.</term>
       <term name="chapter" form="short">chap. </term>
+      <term name="scale">Échelle : </term>
```

Comment:
@jjrondeau, when you define translations, you shouldn't include any separating delimiters (like " : " in this case). So you'd want to replace `<term name="scale">Échelle : </term>` by `<term name="scale">Échelle</term>`.

It also looks like you currently don't use this term yet. Instead of

```xml
<text variable="scale" prefix=". Échelle : "/>
```

it would be better to use:

```xml
<group prefix=". " delimiter=" : ">
  <text term="scale"/>
  <text variable="scale"/>
</group>
```

(the group here prevents the "scale" term from being printed when the "scale" variable is empty)

Reply:
Yes, you're right. I will correct this.